### DISCUSSION TITLE
Support for `filtered` views

### DISCUSSION THREAD

**Discussion by wochinge** [avisi-cloud/structurizr-site-generatr#278] [Q&A] [answered]
Hey y'all! 👋🏻 

Is there currently any way to display [filtered views](https://github.com/structurizr/dsl/blob/master/docs/language-reference.md#filtered-view)?

**Comment by dirkgroot** [comment, marked as answer]
Hi @wochinge. Filtered views are not included in the site by default. You can, however, include them by embedding them in a Markdown file (included by either the `!docs` or the `!adrs` directive).

See https://avisi-cloud.github.io/structurizr-site-generatr/main/embedding-diagrams-and-images/ for more information.

Does this answer your question?

**Comment by zhelyan** [comment]
Hi, looks like filtered views aren't exported?
